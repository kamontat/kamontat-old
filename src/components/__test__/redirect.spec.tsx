import React from "react";
import { mount } from "enzyme";

import Redirect from "../redirect";

describe("Redirect component", () => {
  it("render correctly", () => {
    const component = mount(<Redirect url="https://google.com" />);

    expect(component).toMatchSnapshot();
  });
});
