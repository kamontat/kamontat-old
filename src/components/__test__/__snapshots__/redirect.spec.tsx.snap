// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Redirect component render correctly 1`] = `
<ExtRedirect
  second={0}
  url="https://google.com"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
    meta={
      Array [
        Object {
          "content": "0; url=https://google.com",
          "http-equiv": "refresh",
        },
      ]
    }
  >
    <SideEffect(NullComponent)
      defer={true}
      encodeSpecialCharacters={true}
      meta={
        Array [
          Object {
            "content": "0; url=https://google.com",
            "http-equiv": "refresh",
          },
        ]
      }
    >
      <NullComponent
        defer={true}
        encodeSpecialCharacters={true}
        meta={
          Array [
            Object {
              "content": "0; url=https://google.com",
              "http-equiv": "refresh",
            },
          ]
        }
      />
    </SideEffect(NullComponent)>
  </HelmetWrapper>
</ExtRedirect>
`;
